datalad/config.py

```diff
@@ -336,19 +343,14 @@ def reload(self, force=False):
         self._merged_store = merged
 
     def _need_reload(self, store):
-        if not store['mtimes']:
+        storestats = store['stats']
+        if not storestats:
             return True
+
         # we have read files before
         # check if any file we read from has changed
-        current_time = time()
```

Comment:
This was the last use of `time` in the module, so the import could be dropped.

Reply:
thanks, indeed!